lib/os_mon/src/disksup.erl

```diff
@@ -77,8 +77,8 @@ dummy_reply(get_almost_full_threshold) ->
 dummy_reply({set_almost_full_threshold, _}) ->
     ok.
 
-param_type(disk_space_check_interval, Val) when is_integer(Val),
-						Val>=1 -> true;
+param_type(disk_space_check_interval, Val) when is_number(Val),
+						0<Val -> true;
```

Comment:
not sure if this is a concern, but this seems to allow an effective check interval of zero if `Val` is smaller than `1 / 60_000`

Reply:
thanks